src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -265,11 +267,28 @@ pub enum Connector {
     },
     S3 {
         bucket: String,
+        /// The list of places that we will look for object keys
```

Comment:
Per our last discussion (https://github.com/MaterializeInc/materialize/pull/5194#discussion_r551560634) I think this should probably read something like

```
/// The key sources specified in the `OBJECTS FROM` clause.
```